programs/gateway/src/instructions/update_voter_weight_record.rs

```diff
@@ -41,6 +50,8 @@ pub fn update_voter_weight_record(
         &ctx.accounts.registrar.gatekeeper_network,
         None
     ).or(Err(error!(GatewayError::InvalidGatewayToken)))?;
+    
```

Comment:
We don't need `voter_weight_action` and `target` params for the instruction because the `civic' plugin provided weight is not specific to any action or target

Reply:
✅